### ISSUE TITLE
How to handle case of variable names?

### ISSUE THREAD

**Issue by Felienne** [Felienne/hedy#169]
A kids tried this:

```
naam is Hedy
print 'ik heet ' Naam
```

What to do? Do we force all variables names lowercase? Do we catch and give a specific error message? It is not common but it is sensible to expect this to work as if it is the same name.

**Comment by fpereiro** [comment]
I am inclined to suggest to give them a specific error message. Paying attention to case is essential, and at the same time something that shouldn't be too hard for students with that level of literacy.

An alternative would be to introduce case-sensitivity as part of a level, or an entire level per se. But I like the first one more.

**Comment by Felienne** [comment]
Yeah I was thinking it could be a level too, but I think for now we can leave this open. It could be a BSc project also to build and try alternatives!

**Comment by MarinusVanDijk** [comment]
Perhaps you can consider the use of a phonetic comparison algorithm!

https://en.wikipedia.org/wiki/Phonetic_algorithm

In that case variable misspelling can be corrected.
